.github/workflows/ci.yml

```diff
@@ -5,47 +5,42 @@ on:
     branches:
       - master
   pull_request:
-    branches:
-      - "*"
 
 jobs:
   test:
     name: Test
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-      - uses: actions-rs/cargo@v1
-        with:
-          command: test
+      - uses: dtolnay/rust-toolchain@nightly
```

Comment:
Does this need the `nightly`?

Reply:
no, this one can be stable (though being nightly also does not hurt). Would you prefer me to revert this to stable?